packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -211,47 +212,68 @@ const prompts = () => {
         s === '' ||
         defaultEinsteinAPIError
 
+    const demoChoice = 'Yes'
+    const customChoice = 'No, use custom settings'
+
     const questions = [
+        {
+            name: 'useDemoSettings',
+            message: 'Do·you·want·to·try·out·PWA·Kit·with·the·demo·storefront?',
```

Comment:
Hmm, not sure why there are these dots in the message...

```suggestion
            message: 'Do you want to try out PWA Kit with the demo storefront?',
```

Reply:
I'm not sure how those got in there either. Merging in your change to clean those dots up.